### DISCUSSION TITLE
What is open-props and how does it relate to argyleink/open-props?

### DISCUSSION THREAD

**Discussion by hyperknot** [facebook/stylex#783] [General]
What is open-props and how does it relate to [argyleink/open-props?](https://github.com/argyleink/open-props)?

I mean I opened the package source and it looks like some kind of a design-system default values, but I couldn't find anything about how would I use it.

**Comment by nmn** [comment]
`open-props` is a translation of [argyleink/open-props?](https://github.com/argyleink/open-props) to StyleX syntax. It's the same set of values.

See `apps/nextjs-example` to see how to use it.

**Reply by hyperknot** [reply]
So it's basically just a curated set of values. I don't quite understand the 4.8k GitHub stars on it, but I think I get what it does.

**Reply by nmn** [reply]
Yes, it's a good set of defaults.
